### PR TITLE
#2 Rename roadmap to sketchmap

### PR DESCRIPTION
**Pull request by abheepsapattnaik**

### CODE REVIEW

**Comment by abheepsapattnaik** [comment]
@kaosdev please review the changes

File: README.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ const arrows = [
 ];
 
 // Instantiate a new roadmap
```
Comment (kaosdev) [inline README.md]: Missed one 😄
```suggestion
// Instantiate a new sketchmap
```

**Comment by kaosdev** [comment]
Oh yep, if you can, we have some references of Roadmap in the examples folder. We should change them also or the examples will not compile.

**Comment by abheepsapattnaik** [comment]
Oh sure..will do that

**Comment by kaosdev** [comment]
Thanks for contributing!

**Pull request merged**